enterprise/backend/src/metabase_enterprise/sandbox/query_processor/middleware/row_level_restrictions.clj

```diff
@@ -129,14 +129,15 @@
                       {:source-query source-query}
                       e)))))
 
-(defn- card-gtap->source
+(mu/defn ^:private card-gtap->source :- mbql.s/SourceQuery
```

Comment:
While debugging some test failures I added more schemas around a lot of stuff to catch logic errors more quickly and make things easier to debug